www-src/vendor/scripts/moment.js

```diff
@@ -0,0 +1 @@
+../../node_modules/moment-timezone/node_modules/moment/moment.js
```

Comment:
Both will work. And neither are clean. 
The true dependency is against moment-timezone, as we have to build the good time from a timezone + a ambiguous time.

But if you prefer, I can add moment.js as a dependency of the project, and put the hacky link as you propose.